frontend/src/MakerApp.tsx

```diff
@@ -1,3 +1,4 @@
+import { ChevronUpIcon, RepeatIcon } from "@chakra-ui/icons";
```

Comment:
Nice, github action is pointing you out what's wrong :)

Reply:
mean though, I ran `yarn run eslint` locally and did not get that error 😅
I'm running the same yarn version... potentially different node version problem :/ ?